### ISSUE TITLE
transfer learning tasklist

### ISSUE THREAD

**Issue by cpadavis** [cpadavis/billy#32]
We should be good to go to do the training of the spacewarps and galzoo datasets on the corrected alexnet architecture. task list is as follows:

- [ ] make sure I didn't break anything with adding overlap_max_pool and changing the layers a bit
- [ ] iterate the following until pleased:
   - run cpd_submit.sh (submits 50 random hyperparameter search jobs for both galzoo and spacewarps to sherlock gpu cluster)
   - using interpret_random_hyperparameter.py, update the regularization and learning rate regions in both galzoo and sw_random_hyperparameter.py
- [ ] write script to further train best galzoo / sw networks
- [ ] flesh out tl_galzoo_to_sw.py
- [ ] take the best galzoo network, and run tl_galzoo_to_sw.py with it to transfer learn
- [ ] write code to analyze networks on the test data
- [ ] analyze networks
- [ ] write paper
- [x] drink beer

**Comment by cpadavis** [comment]
turning this into a collection of links to other issues

**Comment by drphilmarshall** [comment]
Nice! This is an example of an "Epic" - it's good to label the sub-issues the same way, and also label this issue as an epic. I'll show you what I mean.
